ARM64 builds breaking in CI with .NetNative error

### Problem Description

From [CI](https://dev.azure.com/ms/react-native-windows/_build/results?buildId=392835&view=logs): 

```
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.TestPlatform.Utilities'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.TestPlatform.Utilities'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.Common'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.Common'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.ObjectModel'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.ObjectModel'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.TestPlatform.Utilities'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.TestPlatform.Utilities'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.Common'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.Common'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.ObjectModel'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.ObjectModel'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.TestPlatform.Utilities'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.TestPlatform.Utilities'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.Common'.
     5>D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.Common'. [D:\a\_work\1\s\vnext\Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj]
##[error]D:\a\_work\_temp\NuGetPackages\microsoft.net.native.compiler\2.2.7-rel-27913-00\tools\Microsoft.NetNative.targets(801,5): Error : ILTransform : warning ILT0010: Could not find assembly 'System.Xml.XPath.XmlDocument, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' referenced by 'Microsoft.VisualStudio.TestPlatform.ObjectModel'.
```

### Steps To Reproduce

Build Arm64 Debug

### Expected Results

_No response_

### CLI version

npx react-native --version

### Environment

```markdown
npx react-native info
```


### Target Platform Version

_No response_

### Target Device(s)

_No response_

### Visual Studio Version

_No response_

### Build Configuration

_No response_

### Snack, code example, screenshot, or link to a repository

_No response_

--- a/change/react-native-windows-8a89df6e-f669-46dc-9ada-d645db4f038d.json
+++ b/change/react-native-windows-8a89df6e-f669-46dc-9ada-d645db4f038d.json
@@ -1,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Remove Microsoft.ReactNative.Managed.IntegrationTests from ARM64 build",
+  "packageName": "react-native-windows",
+  "email": "jthysell@microsoft.com",
+  "dependentChangeType": "patch"
+}

--- a/vnext/Microsoft.ReactNative.Managed.IntegrationTests/Microsoft.ReactNative.Managed.IntegrationTests.csproj
+++ b/vnext/Microsoft.ReactNative.Managed.IntegrationTests/Microsoft.ReactNative.Managed.IntegrationTests.csproj
@@ -49,7 +49,6 @@
     <PlatformTarget>ARM64</PlatformTarget>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>true</Prefer32Bit>
     <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|ARM64'">
@@ -60,7 +59,6 @@
     <PlatformTarget>ARM64</PlatformTarget>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>true</Prefer32Bit>
     <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x64'">
@@ -71,7 +69,6 @@
     <PlatformTarget>x64</PlatformTarget>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>true</Prefer32Bit>
     <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|x64'">
@@ -82,7 +79,6 @@
     <PlatformTarget>x64</PlatformTarget>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
-    <Prefer32Bit>true</Prefer32Bit>
     <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
   </PropertyGroup>
   <PropertyGroup>

--- a/vnext/Microsoft.ReactNative.sln
+++ b/vnext/Microsoft.ReactNative.sln
@@ -123,9 +123,9 @@ Global
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
 		Debug|ARM64 = Debug|ARM64
-		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
+		Release|ARM64 = Release|ARM64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{F7D32BD0-2749-483E-9A0D-1635EF7E3136}.Debug|ARM64.ActiveCfg = Debug|ARM64

--- a/vnext/Microsoft.ReactNative.sln
+++ b/vnext/Microsoft.ReactNative.sln
@@ -119,31 +119,13 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.ReactNative.Managed.IntegrationTests", "Microsoft.ReactNative.Managed.IntegrationTests\Microsoft.ReactNative.Managed.IntegrationTests.csproj", "{E2BE6630-21C7-43A4-AC90-8D2844C06617}"
 EndProject
 Global
-	GlobalSection(SharedMSBuildProjectFiles) = preSolution
-		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{14fa0516-e6d7-4e4d-b097-1470198c5072}*SharedItemsImports = 4
-		Mso\Mso.vcxitems*{14fa0516-e6d7-4e4d-b097-1470198c5072}*SharedItemsImports = 4
-		Mso\Mso.vcxitems*{1958ceaa-fbe0-44e3-8a99-90ad85531ffe}*SharedItemsImports = 4
-		Shared\Shared.vcxitems*{2049dbe9-8d13-42c9-ae4b-413ae38fffd0}*SharedItemsImports = 9
-		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{6c60e295-c8ca-4dc5-b8be-09888f58b249}*SharedItemsImports = 4
-		Mso\Mso.vcxitems*{6c60e295-c8ca-4dc5-b8be-09888f58b249}*SharedItemsImports = 4
-		Mso\Mso.vcxitems*{84e05bfa-cbaf-4f0d-bfb6-4ce85742a57e}*SharedItemsImports = 9
-		Chakra\Chakra.vcxitems*{93792779-4948-4a5d-8ca7-86ed5e3bec27}*SharedItemsImports = 4
-		Mso\Mso.vcxitems*{93792779-4948-4a5d-8ca7-86ed5e3bec27}*SharedItemsImports = 4
-		Chakra\Chakra.vcxitems*{c38970c0-5fbf-4d69-90d8-cbac225ae895}*SharedItemsImports = 9
-		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{da8b35b3-da00-4b02-bde6-6a397b3fd46b}*SharedItemsImports = 9
-		include\Include.vcxitems*{ef074ba1-2d54-4d49-a28e-5e040b47cd2e}*SharedItemsImports = 9
-		Chakra\Chakra.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
-		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
-		Mso\Mso.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
-		Shared\Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
-	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
-		Debug|ARM64 = Debug|ARM64
+		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86
-		Release|ARM64 = Release|ARM64
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{F7D32BD0-2749-483E-9A0D-1635EF7E3136}.Debug|ARM64.ActiveCfg = Debug|ARM64
@@ -296,8 +278,6 @@ Global
 		{ADED4FBE-887D-4271-AF24-F0823BCE7961}.Release|x86.ActiveCfg = Release|x86
 		{ADED4FBE-887D-4271-AF24-F0823BCE7961}.Release|x86.Build.0 = Release|x86
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|ARM64.ActiveCfg = Debug|ARM64
-		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|ARM64.Build.0 = Debug|ARM64
-		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|ARM64.Deploy.0 = Debug|ARM64
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|x64.ActiveCfg = Debug|x64
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|x64.Build.0 = Debug|x64
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|x64.Deploy.0 = Debug|x64
@@ -305,8 +285,6 @@ Global
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|x86.Build.0 = Debug|x86
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Debug|x86.Deploy.0 = Debug|x86
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Release|ARM64.ActiveCfg = Release|ARM64
-		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Release|ARM64.Build.0 = Release|ARM64
-		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Release|ARM64.Deploy.0 = Release|ARM64
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Release|x64.ActiveCfg = Release|x64
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Release|x64.Build.0 = Release|x64
 		{E2BE6630-21C7-43A4-AC90-8D2844C06617}.Release|x64.Deploy.0 = Release|x64
@@ -341,5 +319,23 @@ Global
 	EndGlobalSection
 	GlobalSection(ExtensibilityGlobals) = postSolution
 		SolutionGuid = {988F03E6-640A-4C7E-8B55-C1291B70669E}
+	EndGlobalSection
+	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{14fa0516-e6d7-4e4d-b097-1470198c5072}*SharedItemsImports = 4
+		Mso\Mso.vcxitems*{14fa0516-e6d7-4e4d-b097-1470198c5072}*SharedItemsImports = 4
+		Mso\Mso.vcxitems*{1958ceaa-fbe0-44e3-8a99-90ad85531ffe}*SharedItemsImports = 4
+		Shared\Shared.vcxitems*{2049dbe9-8d13-42c9-ae4b-413ae38fffd0}*SharedItemsImports = 9
+		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{6c60e295-c8ca-4dc5-b8be-09888f58b249}*SharedItemsImports = 4
+		Mso\Mso.vcxitems*{6c60e295-c8ca-4dc5-b8be-09888f58b249}*SharedItemsImports = 4
+		Mso\Mso.vcxitems*{84e05bfa-cbaf-4f0d-bfb6-4ce85742a57e}*SharedItemsImports = 9
+		Chakra\Chakra.vcxitems*{93792779-4948-4a5d-8ca7-86ed5e3bec27}*SharedItemsImports = 4
+		Mso\Mso.vcxitems*{93792779-4948-4a5d-8ca7-86ed5e3bec27}*SharedItemsImports = 4
+		Chakra\Chakra.vcxitems*{c38970c0-5fbf-4d69-90d8-cbac225ae895}*SharedItemsImports = 9
+		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{da8b35b3-da00-4b02-bde6-6a397b3fd46b}*SharedItemsImports = 9
+		include\Include.vcxitems*{ef074ba1-2d54-4d49-a28e-5e040b47cd2e}*SharedItemsImports = 9
+		Chakra\Chakra.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
+		Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
+		Mso\Mso.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
+		Shared\Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
 	EndGlobalSection
 EndGlobal

--- a/vnext/Microsoft.ReactNative.sln
+++ b/vnext/Microsoft.ReactNative.sln
@@ -120,7 +120,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.ReactNative.Manag
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
 		Debug|ARM64 = Debug|ARM64

--- a/vnext/Microsoft.ReactNative.sln
+++ b/vnext/Microsoft.ReactNative.sln
@@ -123,6 +123,7 @@ Global
 		Debug|ARM64 = Debug|ARM64
 		Debug|x64 = Debug|x64
 		Debug|x86 = Debug|x86
+		Debug|ARM64 = Debug|ARM64
 		Release|ARM64 = Release|ARM64
 		Release|x64 = Release|x64
 		Release|x86 = Release|x86

